assets/src/stories-editor/components/higher-order/with-right-click-handler.js

```diff
@@ -38,6 +42,16 @@ const applyWithSelect = withSelect( ( select, props ) => {
 			return;
 		}
 
+		// Ignore if it's a keydown event and the correct combo hasn't been pressed.
```

Comment:
Can we document the right combo here in a comment? Just so it's more clear.

From what I've seen in a quick search, <kbd><kbd>Shift</kbd>+<kbd>F10</kbd></kbd> seems to be a Windows thing, and macOS doesn't even support any keyboard shortcut for that. Bummer.

Reply:
Of course.

Should we try to feature detect and only add the key handler in case it's not supported natively? Also, how to check for that? I'll investigate...

Reply:
I don't think we can detect this. Sending simulated keyboard events won't trigger the context menu and you can't query about potential keys on the keyboard (most win keyboards have the "menu" key, that opens this menu just like Shift+F10.

We can detect OS, but that's a bit crude.

Reply:
Don't think we need to feature detect it. If the OS already supports it, we're not changing behavior, just re-implementing it.

Reply:
Made irrelevant in 3fa7dd1.